.github/workflows/main-build.yml

```diff
@@ -40,6 +40,9 @@ jobs:
         with:
           path: ~/.sonar/cache/
           key: ${{ runner.os }}-sonar
+      - name: Copy module on Java 11
```

Comment:
Hmmm Looks more like not 8 instead of 11 - and it is the module info so I would go with "copy module info" - I think the rest is implicit knowledge by java, and is easy readable via the action